src/main/java/org/opentripplanner/routing/impl/GraphPathFinder.java

```diff
@@ -306,9 +301,7 @@ public GraphPathFinder(Router router) {
                             (options.arriveBy && joinedPath.states.getLast().getTimeInMillis() < options.dateTime * 1000)){
                         joinedPaths.add(joinedPath);
                         if(newPaths.size() > 1){
-                            for (FeedScopedId tripId : joinedPath.getTrips()) {
-                                options.banTrip(tripId);
-                            }
+                            options.banTripsInPath(joinedPath);
```

Comment:
This block is just really poorly commented in general. Not saying we should do a ton to it, but it's honestly hard to know what I'm looking at.

Reply:
Yes, I don't fully understand either, but all of this method wasn't created in the PR and this logic doesn't exist in OTP 2 anyways.